tree/ntuple/src/RFieldBase.cxx

```diff
@@ -122,35 +122,6 @@ void ROOT::RFieldBase::RValue::BindRawPtr(void *rawPtr)
 
 //------------------------------------------------------------------------------
 
-ROOT::RFieldBase::RBulkValues::RBulkValues(RBulkValues &&other)
-   : fField(other.fField),
-     fValueSize(other.fValueSize),
-     fCapacity(other.fCapacity),
-     fSize(other.fSize),
-     fIsAdopted(other.fIsAdopted),
-     fNValidValues(other.fNValidValues),
-     fFirstIndex(other.fFirstIndex)
-{
-   std::swap(fDeleter, other.fDeleter);
-   std::swap(fValues, other.fValues);
```

Comment:
I think this swap is necessary, otherwise the pointer may just get copied and we'd end up freeing the values twice in the destructor, no?